### DISCUSSION TITLE
[HW4] Prob 1 missing

### DISCUSSION THREAD

**Discussion by oglee815** [seominjoon/kaist-ai605#55] [Q&A] [answered]
I found that the problem 1 is missing in HW4.
Is it okay?

**Comment by seominjoon** [comment, marked as answer]
You are right, there is no problem in section 1.

**Reply by JinkyungJo** [reply]
Then, the score including bonus points is out of 130 points?

**Reply by seominjoon** [reply]
Oops.. My bad. I added a really easy "Problem 1.1"  so check it out!

**Reply by seominjoon** [reply]
it is 10 points so now 140 points altogether.
